pkg/common/functions.go

```diff
@@ -0,0 +1,62 @@
+package common
```

Comment:
Generally the file name would be the same name of the package, in this case, `common.go`. When we start having more related to context, json, etc... we could split them into `context.go`, `json-utils.go`. etc..

Reply:
We've already started having more. I'll try to be more domain domain-oriented than type-oriented then. Thanks.